src/lib/analytics/analytics.tsx

```diff
@@ -52,9 +52,7 @@ export function init(store: RootStoreModel) {
     const sess = store.session.currentSession
     if (sess && sess.did) {
       if (sess.email) {
```

Comment:
Still checking for email here and in the other snippet, is that needed?

Reply:
Fixed.